src/app/frank/fd_frank_quic.c

```diff
@@ -19,6 +20,12 @@ init( fd_frank_args_t * args ) {
   args->xsk = fd_xsk_join( fd_wksp_pod_map( args->tile_pod, "xsk" ) );
   if( FD_UNLIKELY( !args->xsk ) ) FD_LOG_ERR(( "fd_xsk_join failed" ));
 
+  /* We close utility.c:namespace_original_fd which is opened with enter_network_namespace and
+     closed in leave_network_namespace. This tile does not want to leave the namespace so close the FD
+     instead of leaking it. */
+  int ret = close( 4 );
```

Comment:
This feels a bit hacky and modifies the prod boot sequence to deal with development code, which I don't think we want. If you launch the Solana process before any Firedancer tiles, you can do the following sequence.

 1. Leave netns if it's entered (due to running config), this only needs to happen in `dev.c` as fdctl will never do configuration as part of `run`, close the fd.
 2. Spawn Solana tile. Not in network namespace. No fds open.
 3. Rejoin netns from path /var/... if needed.
 4. Close after joining.
 5. Spawn Firedancer tiles, no fds open.

Reply:
updated